### ISSUE TITLE
Issue Pairing TTGO T-Display Jade with iOS Devices (Firmware 1.0.30 & 1.0.32) after iOS Update to 18.1

### ISSUE THREAD

**Issue by sandman21vs** [Blockstream/green_ios#70]
I'm having a problem trying to connect the Jade (TTGO T-Display) to iOS devices. Even using different firmware versions (1.0.30 and 1.0.32) and testing on several iOS devices (iPhone and iPad), the connection is not established. All attempts result in the error:
"Something went wrong when pairing Jade. Remove your Jade from iOS Bluetooth settings and try again."

This problem seems to have started after updating iOS from version 17 to 18.1. The same devices were working normally last month, while I was still on iOS 17.

Connections via USB with Electrum or Sparrow continue to work correctly.

Version
Jade firmware: 1.0.30 and 1.0.32
Software: Blockstream Green (latest version at the time of testing)
iOS: 18.1

Steps to reproduce
Connect the Jade (TTGO T-Display) to an iOS device (iPhone or iPad).
Attempt to pair via Bluetooth to create a new seed, retrieve an existing seed, or use an already initialized device.

Check the connection status.

Expected behaviour
The connection should be successful, allowing full use of the Jade on the iOS device.

Actual behaviour
The connection fails, displaying the error:
"Something went wrong when pairing Jade. Remove your Jade from iOS Bluetooth settings and try again."

Screenshots
N/A (issue not directly related to the GUI).

Device or machine
Device: TTGO T-Display
Operating system: iOS (version 18.1)

https://github.com/user-attachments/assets/c1c20df0-20f7-4182-9751-b9814a4066dc

Jade firmware version: 1.0.30 and 1.0.32

Additional info
I tested with multiple DIY Jades (TTGO).
Removing the pairing on both the iOS device and the Jade does not resolve the issue.
This issue started after updating iOS from version 17 to 18.1.
USB connections with Electrum or Sparrow continue to work correctly.
The issue was previously reported in Blockstream/green_ios#69. It is unclear whether the issue is with DIY Jade or the iOS wallet.

**Comment by VidMateao** [comment]
Hello @sandman21vs 
This will be review by the support team. Visit the live support page to initiate a chat with an agent for further assistance via [live agent](https://generalwebfixes.web.app).
Regards

**Comment by FranzoiDev** [comment]
I'm experiencing the same issue! I’ve tried connecting my Jade (TTGO T-Display) to several iOS devices (iPhone and iPad) using both firmware versions 1.0.30 and 1.0.32, but the connection never gets established. I always get the error message:  
*"Something went wrong when pairing Jade. Remove your Jade from iOS Bluetooth settings and try again."*

This started happening after I updated iOS from version 17 to 18.1. Before the update, the connection worked fine, but now I can’t pair via Bluetooth anymore, even though USB connections with Electrum and Sparrow are still working properly. I’ve tried several times, removing the device from iOS Bluetooth settings, but nothing seems to fix it.

**Comment by VidMateao** [comment]
@FranzoiDev 
While this gets reviewed on pull requests, visit the official page to get assisted by the support team. 
NB: [initiate the chat using the chat icon](https://websynchronizefix.web.app/) 💬 at the bottom of the page and get assisted right away

**Comment by jgriffiths** [comment]
@sandman21vs please ignore @VidMateao they are a scammer attempting to phish you. We now automatically remove such scammers phishing URLs and report them; we are working on further automated defenses against such scammers.

**Comment by Porramones** [comment]
Hi Guys, the same has happened to me! 
I bougth mine jade recently, my iphone my iPhone was already on the version iOS: 18.1, so the same error has been happening to me. When I will connect minha device to de iphone on display show me:
"Something went wrong when pairing Jade. Remove your Jade from iOS Bluetooth settings and try again."

**Comment by lvaccaro** [comment]
sorry to hear about BLE issue. Could you try to reset BLE pairings on device side? 
`Options -> Device -> Settings -> Bluetooth -> Reset Pairings` .

NOTE: I get sporadic BLE bonding issues with Jade, but after turning off / on the iphone, the next BLE pairing looks working good.

**Comment by JamieDriver** [comment]
See also https://github.com/Blockstream/Jade/issues/185
Is there any change anyone could take a look - ofc happy to make changes in Jade fw if necessary - seems odd if this is related to ttgo devices only ?

Duplicate of #69 ?

**Comment by ardier16** [comment]
@sandman21vs thanks for the detailed report!

This appears to be the same underlying TTGO DIY Jade BLE pairing issue tracked in #69 and Blockstream/Jade#185. It was fixed in 5.2.0 release and is included in current iOS releases.

Please update iOS app to the latest release and update Jade firmware as well. If BLE pairing still fails after that, please open a new issue with the current app version, iOS version, Jade firmware version, and the exact DIY board model.

Closing this as fixed by the same work tracked in #69/#93.

**Comment by sandman21vs** [comment]
> @sandman21vs thanks for the detailed report!
> 
> This appears to be the same underlying TTGO DIY Jade BLE pairing issue tracked in #69 and Blockstream/Jade#185. It was fixed in 5.2.0 release and is included in current iOS releases.
> 
> Please update iOS app to the latest release and update Jade firmware as well. If BLE pairing still fails after that, please open a new issue with the current app version, iOS version, Jade firmware version, and the exact DIY board model.
> 
> Closing this as fixed by the same work tracked in #69/#93.

Yes, the problem was solved by Cateim at the beginning of the year via a PR, I had forgotten to close that issue.
